HiP-DataStore.Model/Entity/Review.cs

```diff
@@ -48,10 +39,10 @@ public ReviewUpdateArgs CreateReviewUpdateArgs()
 
         public bool Approved { get; set; }
 
-        // Type of the entity the review belongs tos
+        // Type of the entity the review belongs to
         public string EntityType { get; set; }
 
-        // Id of the entity the review belongs to
+        // ID of the entity the review belongs to
```

Comment:
Currently there is a problem with int properties that no PropertyChanged event will be created for id 0. This is exactly what my [PR](https://github.com/HiP-App/HiP-DataStore/pull/124) adresses, so my PR should probably get merged first